griptape/structures/structure.py

```diff
@@ -164,9 +167,6 @@ def default_config(self) -> BaseStructureConfig:
 
             embedding_driver = OpenAiEmbeddingDriver() if self.embedding_driver is None else self.embedding_driver
 
-            if self.stream is not None:
```

Comment:
Remove `self.stream is not None` check from `if self.prompt_driver is not None or self.embedding_driver is not None or self.stream is not None:`

Reply:
done